llama_stack/providers/inline/agents/meta_reference/agent_instance.py

```diff
@@ -144,87 +144,91 @@ def turn_to_messages(self, turn: Turn) -> List[Message]:
     async def create_session(self, name: str) -> str:
         return await self.storage.create_session(name)
 
-    @tracing.span("create_and_execute_turn")
     async def create_and_execute_turn(
```

Comment:
I think this should have resulted in two spans (one a child of another) with the same name. but it seems like it did not?